internal/common/lib.rs

```diff
@@ -11,3 +11,31 @@ pub mod key_codes;
 
 #[cfg(feature = "shared-fontdb")]
 pub mod sharedfontdb;
+
+/// Detect the native style depending on the platform
+pub fn get_native_style(has_qt: bool, target: &str) -> &'static str {
+    // NOTE: duplicated in api/cpp/CMakeLists.txt
+    if target.contains("android") {
+        "material"
+    } else if target.contains("windows") {
+        "fluent"
+    } else if target.contains("apple") {
+        "cupertino"
+    } else if target.contains("wasm") {
+        "fluent"
+    } else if target.contains("linux") | target.contains("bsd") {
+        if has_qt { "qt" } else { "fluent" }.into()
```

Comment:
Instead of `TARGET`, wouldn't it perhaps be better to check for `CARGO_CFG_TARGET_FAMILY` being `unix`? That way it also covers QNX for example. And the same can be used to check for Windows.

Reply:
But it is unix for mac and android as well.  
So I would have to pass two env variables :-/

Reply:
Right, only because of Android :(

Reply:
I mean, leave it if you prefer this way. I think checking the target family is a little cleaner, but it doesn't really matter since this is internals anyway :)